### PR TITLE
Prisma upgrade

### PR DESCRIPTION
**Pull request by paazmaya**
Replacement for #204

---
As per suggestions at the Telegram chat group, the existing migration files have been removed to make it possible to upgrade Prisma.

### CODE REVIEW

**Comment by paazmaya** [comment]
Ran locally with https://github.com/nektos/act and got the following output:

> prisma:warn Prisma failed to detect the libssl/openssl version to use, and may not work as expected. Defaulting to "openssl-1.1.x".
| Please manually install OpenSSL via `apt-get update -y && apt-get install -y openssl` and try installing Prisma again. If you're running Prisma on Docker, add this command to your Dockerfile, or switch to an image that already has OpenSSL installed.

I could try exactly that, what is suggested.

**Comment by paazmaya** [comment]
Finally going all the way to the `e2e` tests where the tests are failing.

File: prisma/entrypoint.sh [hunk 1]
```diff
@@ -9,7 +9,8 @@ echo Start applying migrations...
 npx ts-node manual_migrations.ts
 
 # apply migration
-npx prisma migrate up -c --auto-approve --experimental
+npx prisma db push
+npx prisma migrate dev
```
Comment (pashidlos) [inline prisma/entrypoint.sh:13]: seems like `migrate dev` is for development use only 
could we use `seed` instead? https://www.prisma.io/docs/reference/api-reference/command-reference#db-seed

File: README.md [hunk 2]
```diff
@@ -8,12 +8,13 @@
 - Install Node `14`
```
Comment (pashidlos) [inline README.md:8]: finally we could use `18` :)

File: README.md [hunk 3]
```diff
@@ -8,12 +8,13 @@
...
 - clone repo
 - Update `.env` and `prisma/.env`
-- Make sure Postgres is up and running
+- Make sure Postgres is up and running, using `docker-compose up` in a separate terminal
 - `npm i`
 - `npm run test`
-- Create DB structure and apply migrations `npx prisma migrate up -c --experimental`
+- Create DB structure `npx prisma db push`
+- Apply migrations`npx prisma migrate dev`
```
Comment (pashidlos) [inline README.md:15]: this is needed only in case schema changes 
otherwise `deploy`

File: prisma/entrypoint.sh (hunk 1 above)
Comment (paazmaya) [reply prisma/entrypoint.sh]: Changed

**Comment by pashidlos** [comment]
made RC for testing purposes 
https://github.com/Visual-Regression-Tracker/backend/releases/tag/api-v5.0.0-rc1

**Comment by pashidlos** [comment]
@paazmaya  having some difficulties with migration are not applied 
as well as postgress version conflict 
gonna try solving them and making migration notes for future use

**Pull request merged**